components/brave_rewards/resources/rewards_panel/lib/extension_api_adapter.ts

```diff
@@ -97,13 +97,13 @@ export function getRewardsParameters () {
 
 const externalWalletLoginURLs = new Map<ExternalWalletProvider, string>()
 
-export function getExternalWalletLoginURL (provider: ExternalWalletProvider) {
-  return new Promise<string>((resolve) => {
+export async function getExternalWalletLoginURL (provider: ExternalWalletProvider) {
```

Comment:
Curious: does es-lint require this change?

Reply:
yes: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/promise-function-async.md

Looks like this has been auto fixed. We're negotiating some rules to be turned off.

Reply:
We have turned this rule off for now. We'll address it on another PR.